templates/server/src/main/resources/archetype-resources/server/pom.xml

```diff
@@ -112,6 +108,7 @@
             <configuration>
               <packagingExcludes>WEB-INF/classes/config/application.properties</packagingExcludes>
               <warName>${project.artifactId}</warName>
+              <failOnMissingWebXml>false</failOnMissingWebXml>
```

Comment:
I do not get why this had to be added? Did we downgrade the `maven-war-plugin` or what?
In such case it is always very helpful to add an XML comment with an hyperlink or a rationale.

Reply:
Effective version of `maven-war-plugin` now is `3.2.0` but that version was also used in the archetype as released before. I am quite sure there is a good reason for this change but I can not get it.